rainbow/robust.js

```diff
@@ -37,18 +37,25 @@ class RobustWeb3 {
         if (e && e.toString() === 'Error: connection not open') {
           this.web3.setProvider(this.ethNodeUrl)
         }
+        throw e
       }
     })
   }
 
   async getBlock(b) {
     return await backoff(RETRY, async () => {
       try {
-        return await this.web3.eth.getBlock(b)
+        let block = await this.web3.eth.getBlock(b)
```

Comment:
I am not sure I understand why we have to re-throw here and everywhere else.  Wy don't we return only when `block !== null` and when `block === null` sleep for some time.

Reply:
We must re-throw the error otherwise backoff will not retry. backoff encapsulate the pattern to sleep and retry when the inner function throws error. There is no need to manually sleep

Reply:
Why do we rethrow here: https://github.com/near/rainbow-bridge-lib/pull/15/files#diff-8376a01073a3b2d04e08c11bd16ac7fbR159 , https://github.com/near/rainbow-bridge-lib/pull/15/files#diff-8376a01073a3b2d04e08c11bd16ac7fbR263 , https://github.com/near/rainbow-bridge-lib/pull/15/files#diff-8376a01073a3b2d04e08c11bd16ac7fbR283